src/libs/antares/correlation/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ target_link_libraries(correlation
         yuni-static-core
         inifile
         array
+        PUBLIC
+        ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
to be removed

Reply:
It is necessary to make headers public.
I will update with a "include" folder

Reply:
My bad, mixed up include and link